modules/_labs/combobox/react/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@workday/canvas-kit-labs-react-combobox",
+  "version": "3.0.0-beta.1",
```

Comment:
I thought the version for new modules needed to start at `0.0.0`, but the CI seemed to pass...